src/utils/stats-detection.js

```diff
@@ -253,6 +253,50 @@ function getTotalReceivedPacketsStandard(statsEntry, report) {
     }
 }
 
+
+/**
+ * Return standard statistics for received and lost packets.
+ *
+ * @param {Object} report - Individual stat report.
+ * @param {Object} statsEntry - Complete rtcstats entry
+ * @returns  {VideoSummary|undefined}
+ */
+function getInboundVideoSummaryStandard(statsEntry, report) {
+
+    if (report.type === 'outbound-rtp') {
+        // we ignore outbound video for now.
+        return;
+    }
+
+    if (report.type === 'inbound-rtp' && report.mediaType === 'video') {
+        // Handles google-standard-stats-*
+        return {
+            frameHeight: report.frameHeight,
+            framesPerSecond: report.framesPerSecond
+        };
+    }
+    if (report.type === 'ssrc' && report.googFrameHeightReceived) {
+        // Found in chrome96-standard-stats-p2p-add-transceiver and in google-legacy-*
+        return {
+            frameHeight: report.googFrameHeightReceived,
+            framesPerSecond: report.googFrameRateOutput
+        };
+    }
+
+    if (report.type === 'track' && report.remoteSource === true && report.frameHeight) {
+        // Found in google-standard-stats-* and safari-standard-stats. Unfortunately it contains cumulative frames, so
+        // no rate that we can extract.
+        return {
+            frameHeight: report.frameHeight
+        };
+    }
+
+    if (report.type === 'inbound-rtp' && report.framerateMean) {
+        // Handles frame rate but no resolution in firefox-standard-stats-sfu.
```

Comment:
The comment indicates frame rate handling, but the code doesn't.

Reply:
👍